tests/jballerina-unit-test/src/test/resources/test-src/types/xml/xml-navigation-access.bal

```diff
@@ -170,6 +170,56 @@ function testXMLNavigationWithEscapeCharacter() {
     assert(x7, xml `<home-address>some address</home-address>`);
 }
 
+function testXmlNavigationWithDefaultNamespaceDefinedAfter() {
+    xml[] results = [];
+    xml x1 = xml `<a>a<b>b<c>c</c></b><d>d</d></a><f/>`;
+    results.push(x1.<a>, (x1/*), (x1/<b>), (x1/<*>), (x1/**/<c>));
```

Comment:
Shall we rewrite these with assignments by index instead so that we can directly compare betwen the two functions? 

E.g.,

```ballerina
results[0] = x1.<a>;
```